src/main/java/virtual_pet/VirtualPetApplication.java

```diff
@@ -1,9 +1,100 @@
 package virtual_pet;
 
+import java.util.Scanner;
+
 public class VirtualPetApplication {
+    public static Virtualpet Rick = new Virtualpet(3, 3, 3, 3);
```

Comment:
Should always be placed inside the main().  With it outside of the main() you were forced by the IDE to make it static.  You don't want an instantiation of an object to be static as that defeats the purpose of creating the object when it needs to be used.   Place at line 9.